### DISCUSSION TITLE
Directus Cli Commands to clear to Expired Sessions

### DISCUSSION THREAD

**Discussion by workatease** [directus/directus#4414] [Feature Requests]
We need to have a way to clear the Expired session via CLI
when the users logs in and does not logs out the session persists but it expires but it is never deletes and the entries build up in time.

it would be a good idea to clear it via cli or any other method ( could be better if it is automated )

**Comment by rijkvanzanten** [comment]
> it would be a good idea to clear it via cli or any other method ( could be better if it is automated )

I agree, I think we should delete all sessions where NOW() > expired date automatically 👍🏻

**Reply by workatease** [reply]
what do you think the good place to implement this

**Reply by rijkvanzanten** [reply]
I think we can do it right where we create new sessions too:

https://github.com/directus/directus/blob/e8222d2a289a4c45d7c5753cbd1feda45dc00bc8/api/src/services/authentication.ts#L130

on every creation of a new session, delete everything expired = older than now

**Reply by workatease** [reply]
Its perfect I guess

**Reply by workatease** [reply]
#4504 to close the idea

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
